<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogChangeFieldValuesForGroup</class>
 <widget class="QDialog" name="DialogChangeFieldValuesForGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vaihda ominaisuustietoja joukolle kaavakohteita</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonSelectPolygonFeatures">
       <property name="text">
        <string>Valitse päivitettävät aluekohteet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonChooseUpdatedAttributesAndValuesForSpatialFeatures">
       <property name="text">
        <string>Valitse päivitykset aluekohteille</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonSelectSupplementaryPolygonFeatures">
       <property name="text">
        <string>Valitse päivitettävät täydentävät aluekohteet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonChooseUpdatedAttributesAndValuesForSuplementarySpatialFeatures">
       <property name="text">
        <string>Valitse päivitykset täydentäville aluekohteille</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonSelectLineFeatures">
       <property name="text">
        <string>Valitse päivitettävät viivamaiset kohteet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonChooseUpdatedAttributesAndValuesForSpatialLineFeatures">
       <property name="text">
        <string>Valitse päivitykset viivamaisille kohteille</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButtonSelectPointFeatures">
       <property name="text">
        <string>Valitse päivitettävät pistemäiset kohteet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonChooseUpdatedAttributesAndValuesForSpatialPointFeatures">
       <property name="text">
        <string>Valitse päivitykset pistemäisille kohteille</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Sulje</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
